src/VCF_Tasks.cpp

```diff
@@ -51,4 +51,20 @@ bool run_read_adc2_task(const unsigned long& sysMicros, const HT_TASK::TaskInfo&
 
     return true;
 }
+
+bool init_buzzer_control_task(const unsigned long& sysMicros, const HT_TASK::TaskInfo& taskInfo)
+{
+    pinMode(BUZZER_CONTROL, INPUT);
+
+    return true;
+}
+bool run_buzzer_control_task(const unsigned long& sysMicros, const HT_TASK::TaskInfo& taskInfo)
+{
+    int buzzer_state = digitalRead(BUZZER_CONTROL);
```

Comment:
This needs to happen in reverse. Instead of digitalRead-ing the buzzer control pin and setting the value of system data, you should be digitalWrite-ing the control pin to be the current value of buzzer_is_active.